### DISCUSSION TITLE
使用群晖docker安装后，无法在群晖filestation通过webdav挂载网盘

### DISCUSSION THREAD

**Discussion by Choufish** [AlistGo/alist#848] [Q&A]
按照/webdav的教程，使用ip:port/dav，路径，无法在群晖下挂载网盘

**Comment by damao12322** [comment]
路径前去掉/，仅填写 dav

**Reply by babyedi3** [reply]
thanks

**Reply by azengjing** [reply]
> 路径前去掉/，仅填写 dav

非常感谢，困扰了我好多天的烦恼，轻飘飘的被解决。万分感谢！

**Reply by NovaShen0724** [reply]
Author请确认答案以便他人计算contribution

**Comment by cguo2013** [comment]
我试过用群晖docker 安装alist, 这种情况下确实无法通过群晖的filestation(远程连接的方式)把webdav 挂载到filestation下, 会提示无法访问这个ip. 不知道有没有成功的例子?

**Reply by nionionping** [reply]
alist的docker网络用host
